pyradios/radios.py

```diff
@@ -367,14 +367,30 @@ def stations_by_tag(self, tag, exact=False, **kwargs):
         kwargs.update({"tag": tag, "tag_exact": exact})
         return self.search(**kwargs)
 
+    def stations_by_tagList(self, tagList, **kwargs):
```

Comment:
In order to maintain a coding standard I suggest that we follow these [conventions](https://realpython.com/python-pep8/#naming-styles)

```python
def stations_by_tah_list(self, tag_list, **kwargs):
    tag_list = ",".join(tag_list)
```